brainscore_core/plugin_management/parse_plugin_changes.py

```diff
@@ -1,31 +1,40 @@
-import re
-import subprocess
-import sys
 import json
+import os
 from pathlib import Path
+import re
 from typing import List, Tuple, Dict
 
 from .test_plugins import run_args
 
 PLUGIN_DIRS = ['models', 'benchmarks', 'data', 'metrics']
+SPECIAL_PLUGIN_FILES = ['brainscore_vision/model_interface.py', 'brainscore_language/artificial_subject.py']
 
 
 def separate_plugin_files(files: List[str]) -> Tuple[List[str], List[str]]:
 	"""
-	:return: one list of files that are located inside a plugin, and one list of files that are located outside of all plugins, 
-		e.g. `['models/mymodel/__init__.py', 'models/mymodel/model.py', 'models/mymodel/test.py'], ['model_helpers/make_model_brainlike.py']`
+	:return: one list of files that are located inside a plugin, e.g. `['models/mymodel/__init__.py', 'models/mymodel/model.py', 'models/mymodel/test.py']`,
+		one list of files that are not plugin-related, e.g. `['README.md', 'pyproject.toml']`,
+		and one list of files that are not inside a plugin but will trigger scoring of all related plugins, e.g. `['model_helpers/make_model_brainlike.py', 'data/__init__.py']`
 	"""
 	plugin_files = []
 	non_plugin_files = []
+	plugin_related_files = []
 
 	for f in files:
 		subdir = f.split('/')[1] if len(f.split('/')) > 1 else None
-		if not any(plugin_dir == subdir for plugin_dir in PLUGIN_DIRS):
-			non_plugin_files.append(f)
+		if any(plugin_dir == subdir for plugin_dir in PLUGIN_DIRS):
+			if len(f.split('/')) == 3 and not os.path.isdir(f):
+				plugin_related_files.append(f)
+			else:
+				plugin_files.append(f)
+		elif any(f'{plugin_dir.strip("s")}_helpers' == subdir for plugin_dir in PLUGIN_DIRS):
+			plugin_related_files.append(f)
+		elif any(spf == f for spf in SPECIAL_PLUGIN_FILES):
```

Comment:
```suggestion
		elif any(special_plugin_file == f for special_plugin_file in SPECIAL_PLUGIN_FILES):
```
we can afford the storage costs for the extra characters